packages/app/src/specs/SpecsList.vue

```diff
@@ -102,6 +102,7 @@
           @showConnectToProject="showConnectToProject"
         />
       </div>
+      <div />
```

Comment:
We totally can. My thinking was that on larger widths an additional column gives a bit more padding to center the data a bit more, but this is not entirely necessary.

Reply:
It would probably have to be inner padding or something if we wanted to maintain the column widths right? Not entirely sure on where would be best to put it without misaligning them.

Reply:
Setting right padding here should do it: https://github.com/cypress-io/cypress/blob/peter/feat/aci-columns/packages/app/src/specs/SpecsList.vue#L116

But we'd need to add whatever value we use there to the scrollbarOffset value to keep the column headers aligned: https://github.com/cypress-io/cypress/blob/peter/feat/aci-columns/packages/app/src/specs/SpecsList.vue#L73